src/main/java/org/mozilla/msrp/platform/common/property/VideoProperties.kt

```diff
@@ -0,0 +1,23 @@
+package org.mozilla.msrp.platform.common.property
+
+import org.mozilla.msrp.platform.util.logger
+import org.springframework.boot.context.properties.ConfigurationProperties
+import javax.annotation.PostConstruct
+import javax.inject.Named
+
+/**
+ * Type-safe configuration properties which are setup in application.yml file.
+ */
+@Named
+@ConfigurationProperties("video")
+class VideoProperties {
+    private val log = logger()
+
+    var cacheSize: Long = 100L
```

Comment:
You can just make this value static/global in code. Unless you really want to use `application.yaml` file to handle it in different environments.

Reply:
Fixed